android/autodispose-android/src/main/java/com/uber/autodispose/android/AutoDisposeAndroid.kt

```diff
@@ -15,6 +15,7 @@
  */
 
 @file:Suppress("NOTHING_TO_INLINE")
+@file:JvmName("AutoDisposeAndroid")
```

Comment:
let's nix this and call it KotlinExtensions like the main artifact

Reply:
da380b7f7b50d2c390ab2beab8551683f79c8d28